src/main/java/com/ea/restaurant/grpc/GrpcUserServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.ea.restaurant.grpc;
+
+import com.ea.restaurant.exceptions.EntityNotFoundException;
+import com.ea.restaurant.grpc.userServiceGrpc.userServiceImplBase;
+import com.ea.restaurant.service.UserService;
+import io.grpc.stub.StreamObserver;
+import net.devh.boot.grpc.server.service.GrpcService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@GrpcService
+public class GrpcUserServiceImpl extends userServiceImplBase {
+  @Autowired private UserService userService;
+
+  @Override
+  public void getUserById(id request, StreamObserver<userObject> responseObserver) {
+    var user =
+        userService.findEnabledById(request.getId()).orElseThrow(EntityNotFoundException::new);
```

Comment:
You should use your user controller instead

Reply:
Done.